code/app/src/main/java/com/example/bread/utils/EmotionUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.example.bread.utils; // Ensures proper package placement
```

Comment:
A lot of these comments are not required just have javadoc comment explaining the class